rdrf/registry/patients/admin_forms.py

```diff
@@ -137,6 +137,16 @@ def save(self, commit=True):
             self.instance.filename = self.cleaned_data["form"].name
         return super(PatientConsentFileForm, self).save(commit)
 
+    def clean(self):
```

Comment:
check the meaning of clean
call sequence for forms/models

Reply:
amended